### PR TITLE
DAOS-7390 control: Remove dmg storage prepare

### PR DESCRIPTION
**Pull request by kjacque**
- Remove remote storage prepare command from the dmg tool. Storage
  prepare is still available as a subcommand of the daos_server
  binary, to be run on the local machine.
- Remove dmg storage prepare from functional tests.

Signed-off-by: Kris Jacque <kristin.jacque@intel.com>

### CODE REVIEW

**Review by shimizukko — DISMISSED** [review]
It would be great if you could remove line 9 of dmg_utils.py to fix the pylint issue.

**Review by dinghwah — APPROVED** [review]
LGTM.

**Comment by kjacque** [comment]
The single test failure in CI was [DAOS-7815](https://jira.hpdd.intel.com/browse/DAOS-7815).

File: src/control/go.mod [hunk 1]
```diff
@@ -7,6 +7,7 @@ require (
 	github.com/armon/go-metrics v0.3.6 // indirect
 	github.com/dustin/go-humanize v1.0.0
 	github.com/fatih/color v1.10.0 // indirect
+	github.com/golang/protobuf v1.5.2
```
Comment (tanabarr) [inline src/control/go.mod:10]: this shouldn't be added, stick to the golang.org repo

**Review by tanabarr — CHANGES_REQUESTED** [review]
painful review but looks fine apart from inadvertent go.mod entry

**Review by mjmac — COMMENTED** [review]
LGTM aside from the go.mod change that Tom noted.

File: src/control/go.mod (hunk 1 above)
Comment (kjacque) [reply src/control/go.mod]: Oops, this was added by VSCode trying to be helpful. Removed.

**Review by tanabarr — APPROVED** [review]

**Review by dinghwah — APPROVED** [review]
LGTM.

**Review by mjmac — APPROVED** [review]

**Review by shimizukko — APPROVED** [review]

**Pull request merged**